lib/instrumentation/express-utils.js

```diff
@@ -1,5 +1,6 @@
 'use strict'
 
+var parseUrl = require('parseurl')
```

Comment:
As @Qard suggested in a previous commit, let's drop the dependency to `parseurl` and instead only use it if the user already is using it in their application - and fall back to the regular `url.parse` function in core if not:

```suggestion
var parseUrl
try {
  parseUrl = require('parseurl')
} catch (e) {
  parseUrl = require('url').parse
}
```

This way we get the best of both worlds: If the user isn't already using `parseurl`, we'll not pollute the request object. If they are, they'll probably expect it to happen anyway.

Reply:
so cool 👍

Reply:
I think that our [dependency-check](https://github.com/maxogden/dependency-check) linter will complain if it sees the `parseurl` module being required and not being listed in our package.json dependencies. So we need to ignore that module.

You do that by updating this line:
https://github.com/elastic/apm-agent-nodejs/blob/5a8e06ded78953755b492be5b6f64369be85fe4b/package.json#L16

I think you just need to add `-i parseurl` to the first command, like so:

```
dependency-check . -i async_hooks -i parseurl
```

Reply:
Yep, it didn't like it: https://travis-ci.org/elastic/apm-agent-nodejs/jobs/506371767#L527

Reply:
fixed :) , I learn a lot from this code review